ethnode/files/nginx-geth-endpoint

```diff
@@ -0,0 +1,12 @@
+upstream stream_backend {
+    server 127.0.0.1:8545 max_fails=2 fail_timeout=15s;
+}
```

Comment:
This file is identical to `ethnode/files/sites-available/nginx-geth-endpoint`. Please remove the duplicate.